tox.ini

```diff
@@ -11,8 +11,8 @@ deps =
     mock
     coveralls
 commands =
-    coverage run --source=skt,tests -m unittest discover tests
-    coverage report -m
+    coverage run --branch --source=sktm,tests --omit="*/tests/*" -m pytest -v -s tests
```

Comment:
I think  you have a typo here. It should be `skt` not `sktm`

Reply:
Oh, good catch.